charts/fluid/fluid/templates/webhook/webhookconfiguration.yaml

```diff
@@ -24,4 +24,24 @@ webhooks:
     objectSelector:
       matchLabels:
         serverless.fluid.io/inject: "true"
+  - name: fuse.serverfull.fluid.io
```

Comment:
please rename `serverfull` to `serverful`. Thanks.

Reply:
Done.